app/controllers/application_controller.rb

```diff
@@ -35,6 +35,11 @@ def current_user
     @current_user ||= User.find_by(id: session[:user_id])
   end
 
+  def user_avatar(user)
+    default_avatar = ActionController::Base.helpers.image_path('default-avatar.png')
+    user.avatar.attached? ? url_for(user.avatar) : default_avatar
+  end
```

Comment:
I wouldn't put this here - I would move this to the `User` model instead

Reply:
This method is used in User, Sessions and SharedAccess.
Should we create a User controller concern?
And maybe add current_user in there as well?

Reply:
No - it's an attribute for `user`, so it should be added to the `User` class.

That way whenever you call `user.avatar_with_default` or whatever you call the method, you get the right value returned